src/electionguard/ballot.py

```diff
@@ -414,6 +414,9 @@ class CiphertextBallotContest(ElectionObjectBase, CryptoHashCheckable):
     ballot_selections: List[CiphertextBallotSelection]
     """Collection of ballot selections"""
 
+    ciphertext: ElGamalCiphertext
```

Comment:
We use `ciphertext` for a lot for properties but I don't this is helpful for determining what this is because we use `ciphertext` in so many sections. `accumulated_votes` or `total` would make sense. idk if always need to indicate it's `encrypted` or `ciphertext`